albumentations/augmentations/transforms.py

```diff
@@ -1371,6 +1371,8 @@ class RandomGamma(ImageOnlyTransform):
     Args:
         gamma_limit (float or (float, float)): If gamma_limit is a single float value,
             the range will be (-gamma_limit, gamma_limit). Default: (80, 120).
+        p_invert (float): Probability of applying transform symmetrical to gamma transform with respect to the y=x.
+            Identical to sequentially applied InvertImg, RandomGamma and InvertImg. Default: 0.0.
```

Comment:
I think would be great add there link to the [issue](https://github.com/albumentations-team/albumentations/issues/1308) or to the [article](https://www.cambridgeincolour.com/tutorials/gamma-correction.htm) directly